code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm

```diff
@@ -360,6 +368,30 @@
 	if(volume >= 1)
 		T.wet_floor(2)
 
+/datum/reagent/cardox
+	name = "Cardox"
+	id = "cardox"
+	description = "Cardox is an expensive, NanoTrasen designed cleaner intended to eliminate liquid phoron stains from suits."
+	reagent_state = LIQUID
+	color = "#EEEEEE"
+	taste_description = "cherry"
+
+/datum/reagent/toxin/cardox/affect_touch(var/mob/living/carbon/M, var/alien, var/removed)
+	.. ()
+	if(alien == IS_VAURCA)
```

Comment:
Use isvaurca(M) as is used everywhere else.

Reply:
`isvaurca()` checks should not be used in reagent code.